web/src/selectors/Assertion.selectors.ts

```diff
@@ -17,9 +17,46 @@ const selectAffectedSpanList = createSelector(stateSelector, paramsSelector, (st
   return trace?.spans.filter(({id}) => spanIdList.includes(id)) || [];
 });
 
+const selectResultAssertionsBySpan = createSelector(
+  stateSelector,
+  (_: RootState, testId: string, runId: string, spanId: string) => ({testId, runId, spanId}),
+  (state, {testId, runId, spanId}) => {
+    const {data} = endpoints.getRunById.select({testId, runId})(state);
```

Comment:
We grab the test run only once, and then it is updated using the dry-run, but that is only saved in the TestDefinitionSlice under the assertionResults entry. We should already have a selector to grab it from the store by passing the state so no biggies.

Reply:
@xoscar I don't think we should read from `TestDefinitionSlice`. But let's connect in slack about this.